### ISSUE TITLE
Using Moveit together with Navigation Stack

### ISSUE THREAD

**Issue by adamanov** [ros-planning/moveit#3165]
### Description

Overview of your issue here.

### Your environment
* ROS Distro: [Melodic|
* OS Version: e.g. Ubuntu 18.04
* Robots: Clearpath Jackal mobile platform  + OpenMANIPULATOR-P (RM-P60-RNH)

### Steps to reproduce
I create a combined URDF model for both robot, where Arm placed on the Jackal mobile platform. After that used Moveit Asistant setup, and created a combined moveit_config file for URDF model 
![image](https://user-images.githubusercontent.com/53511072/174647381-344a230d-b4d0-459a-a2e2-84e23f7d7388.png)

### Expected behavior
By launching both `move_group.launch` (from moveit) and `move_base.launch` (from navigation stack) I expected to drive with jackal to the destination point and after that use moveit to pick a certain object. 

### Actual behavior
The robot platform does not move after setting a Goal position from Rviz, however, I can still move the robot arm within moveit rviz window.

I assume that the problem is related to the controller of both robots. But I am not sure if both frameworks can be used together on the single robot (mobile platform with robot arm)

### Question
Is it possible to use both frameworks together? or How can I control both the mobile platforms and arm robots with Rviz? for a future task in gazebo environment?

**Comment by simonschmeisser** [comment]
You triggered my curiosity, how did you solve your problem? Or was it answered on answers.ros.org or similar?

**Comment by adamanov** [comment]
> You triggered my curiosity, how did you solve your problem? Or was it answered on answers.ros.org or similar?

The problem was related gazebo ros_control plugings. It was launched in two urdf file (in jackal urdf and in arm urdf). Therefore one of the controller manager did not work properly. I had to delete duplicated gazebo_ros_control pluging from urdf model. After that the problem was solved
<plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
